content/docs/connect/choose-connection.md

```diff
@@ -0,0 +1,163 @@
+---
+title: Choosing your connection type
+subtitle: Rules of thumb for choosing one connection type over another
+enableTableOfContents: true
+---
+
+When setting up your application’s connection to your Neon Postgres database, you need to make two main choices: the type of driver and the type of connection. This flowchart will guide you through these options.
```

Comment:
```suggestion
When setting up your application’s connection to your Neon Postgres database, you need to make two main choices: 

- **Postgres driver type**: Serverless driver vs. TCP-based driver
- **Connection type**: pooled vs. unpooled

This flowchart will guide you through these selections.
```

Reply:
not sure about "direct" vs "unpooled". We're using both in the docs currently, and the Vercel integration defines a "DATABASE_URL_UNPOOLED" variable. wdyt?

Reply:
Unless it's against industry convention (the terms seem interchangeable from my searches), to me "direct" is clearer. If pooled was the default, so that you have to explicitly disable it, then "unpooled" would make sense since you're making a clear decision to not use pooling.